SPRCLI/Sources/SPR/UpdateList.swift

```diff
@@ -49,7 +49,7 @@ extension SPRPublisher {
         jsonEncoder.outputFormatting = [.prettyPrinted, .sortedKeys]
         let data = try jsonEncoder.encode(list)
         let body = ByteStream.data(data)
-        let input = PutObjectInput(body: body, bucket: bucket, contentType: "application/json", key: listKey)
+        let input = PutObjectInput(body: body, bucket: bucket, cacheControl: "public, no-cache", contentType: "application/json", key: listKey)
```

Comment:
Added a cache-control header here so that SPM will immediately see new published versions as they are posted during development.  I will likely relax this to allow some caching again before we go to production.